deploy/08_configure_keep_registry.ts

```diff
@@ -0,0 +1,26 @@
+import { HardhatRuntimeEnvironment } from "hardhat/types"
+import { DeployFunction } from "hardhat-deploy/types"
+
+const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
+  const { getNamedAccounts, deployments } = hre
+  const { keepDeployer } = await getNamedAccounts()
+  const { execute, log } = deployments
+
+  const TokenStaking = await deployments.get("TokenStaking")
+
+  await execute(
+    "KeepRegistry",
+    { from: keepDeployer },
+    "approveOperatorContract",
+    TokenStaking.address
+  )
+
+  log(
+    `Approved T TokenStaking operator contract [${TokenStaking.address}] in Keep registry`
+  )
+}
+
+export default func
+
+func.tags = ["ConfigureKeepRegistry"]
+func.dependencies = ["TokenStaking", "KeepRegistry"]
```

Comment:
I guess we won't be able to run `approveOperatorContract` from the script for mainnet as it will have to be executed by the governance. Let's add skip for script then:

```ts
func.skip = async function (hre: HardhatRuntimeEnvironment): Promise<boolean> {
  return hre.network.name === "mainnet"
}
```

Reply:
[8e7c5f0](https://github.com/threshold-network/solidity-contracts/pull/78/commits/8e7c5f04fff554e87dd6485563c45ad32568422e)